_inc/lib/tracks/client.php

```diff
@@ -14,17 +14,15 @@
 ```
  */
 
-// Load the client classes
-require_once( dirname(__FILE__) . '/class.tracks-event.php' );
-require_once( dirname(__FILE__) . '/class.tracks-client.php' );
-
 // Now, let's export a sprinkling of syntactic sugar!
 
 /**
  * Procedurally (vs. Object-oriented), track an event object (or flat array)
  * NOTE: Use this only when the simpler jetpack_tracks_record_event() function won't work for you.
  * @param \Jetpack_Tracks_Event $event The event object.
  * @return \Jetpack_Tracks_Event|\WP_Error
+ *
+ * @deprecated 7.5 use Tracking->tracks_record_event_raw instead
  */
 function jetpack_tracks_record_event_raw( $event ) {
 	return Jetpack_Tracks_Client::record_event( $event );
```

Comment:
Let's deprecate each of those functions properly by adding a `_doing_it_wrong` call. We're doing this all over Jetpack for deprecations already.

Also, let's make sure all existing usages of the deprecated functions are replaced with the new ones.

Reply:
Yes!

Reply:
Resolved in d1e7c3d6e4b420eb6521be1b18ee86a9e4a7906a.